src/app/_models/uqj.ts

```diff
@@ -0,0 +1,12 @@
+import {Question} from '@app/_models/question';
+
+export interface UQJ {
+  id: number;
+  random_seed: number;
+  last_viewed: Date;
+  opened_tutorial: boolean;
+  tokens_received: number;
+  is_solved: boolean;
+  is_partially_solved: boolean;
+  question: Question;
```

Comment:
does this work?

Reply:
it actually does (somewhat strangely) should I make an issue to potentially revise the "Question" interface though?